diagnostic.md

```diff
@@ -62,7 +65,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+undefined because b = -5 which doesn't make sense to the console since b > 0
```

Comment:
b > 0 with b = -5 evaluates to false so the entire conditional expression evaluates to false and the if-block doesn't run.  The function then returns a - b, which is 0 - (-5) = 5.